eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2233,6 +2240,22 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
     context.system.eventStream.publish(NetworkFeePaid(self, remoteNodeId, channelId, tx, fee, desc))
   }
 
+  /**
+    * Returns the number of confirmations needed to safely handle the funding transaction,
+    * we make sure the cumulative block reward largely exceeds the channel size.
+    *
+    * @param fundingSatoshis funding amount of the channel
+    * @return number of confirmations needed
+    */
+  def minDepthForFunding(fundingSatoshis: Satoshi): Long = fundingSatoshis match {
+    case funding if funding <= MAX_FUNDING => nodeParams.minDepthBlocks
+    case funding if funding > MAX_FUNDING =>
+      val blockReward = 6.25
```

Comment:
```suggestion
      val blockReward = 6.25 // this is true as of ~May 2020, and will be too conservative after 2024
```

Reply:
This has actually become 
```
// this is true as of ~May 2020, but will be too large after 2024
```
Since the block reward estimate will be too large after 2024 and our min_depth calculations will have to consider this.